tests/api_client/rest_api_client_test.py

```diff
@@ -0,0 +1,74 @@
+from osmaxx.api_client.API_client import RESTApiJWTClient
+from requests_mock import ANY
+
+
+def test_get_requests_combined_url(requests_mock):
+    """
+    The requested URL is the concatenation of
+    the service_base of the RESTApiJWTClient instance and
+    the relative URL passed to RESTApiJWTClient.get().
+    """
+    requests_mock.get(ANY)
+    c = RESTApiJWTClient('http://example.com/service/uri_base/')
+    response = c.get('get/example')
+    assert response.request.url == 'http://example.com/service/uri_base/get/example'
+
+
+def test_get_specifies_body_content_type(requests_mock):
+    # FIXME: Does this make any sense?
+    # As far as I can tell, the 'Content-Type' in the request headers specifies
+    # the content type of the body of the request, not of the body of the response.
+    # While GET requests can have a body (which the server should ignore if present),
+    # we don't send a body in the request. Specifying an absent body's content type
+    # seems rather nonsensical.
+    requests_mock.get(ANY)
+    c = RESTApiJWTClient('http://example.com/service/uri_base/')
+    response = c.get('get/example')
+    assert response.request.headers['Content-Type'] == 'application/json; charset=UTF-8'
+
+
+def test_get_sends_payload_in_query_string(requests_mock):
+    requests_mock.get(ANY)
+    c = RESTApiJWTClient('http://example.com/service/uri_base/')
+    response = c.get('get/example', params={'some_key': b'a value'})
+    assert response.request.url == 'http://example.com/service/uri_base/get/example?some_key=a+value'
```

Comment:
@hixi Until now, we don't actually use this capability. Should the explicit code for it be removed (it would continue to work because we forward `**kwargs`) or should it be retained, so the API client has a somewhat complete interface?